docs/adrs/0004-field-for-institution-information.md

```diff
@@ -0,0 +1,125 @@
+      * **TIMDEX field mapping:** [`repository` -> `publication_information`](https://github.com/MITLibraries/transmogrifier/blob/main/transmogrifier/sources/xml/ead.py#L208-L215)/
+      * **Example value(s):** All records will read "Massachusetts Institute of Technology. Libraries. Department of Distinctive Collections"
+
+   **Transformer:** Marc
+   **Source(s):** MIT Alma (alma)
+   * [`<datafield tag="270">`](https://www.loc.gov/marc/bibliographic/bd270.html): _Not currently used in transformer._ The contents for this data field code describe "An address (as well as electronic access information such as email, telephone, fax, TTY, etc. numbers) for contacts related to the content of the bibliographic item." However, it also reads: 
+      * "For online resources, the addresses may apply to persons or agencies that are responsible for the general availability of the item."
+      * "Contacts related to access to an instance of an online resource at a particular electronic location are contained in field [856 (Electronic Location and access)](https://www.loc.gov/marc/bibliographic/bd856.html) subfield $m (Contact for access assistance)".
+   * [`<datafield tag="260"><subfield code="b">`](https://www.loc.gov/marc/bibliographic/bd260.html): Name of the publisher or distributor.
+   * [`<datafield tag="264"><subfield code="b">`](https://www.loc.gov/marc/bibliographic/bd264.html): Name of producer, publisher, distributor, manufacturer.
+      * **TIMDEX field mapping:** Text from datafield matching the tags and subfield code are joined into a single string value, separated by a single space (`" "`).
+      * **Example value(s):** "Berlin ; Boston : De Gruyter Textbook, [2014]"
+
+   **Transformer:** OaiDc/Springshare OaiDc
+   **Source(s):** Research Databases (researchdatabases), LibGuides (libguides)
+   * [`<dc:publisher>`](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/publisher): An entity responsible for making the resource available.
+      * **TIMDEX field mapping:**: [`publisher` -> `publication_information`](https://github.com/MITLibraries/transmogrifier/blob/main/transmogrifier/sources/xml/oaidc.py#L91-L94)
+      * **Example value(s):** For Research Databases, `publisher` isn't used; for LibGuides, this value will always read "MIT Libraries".
+
+## Decision
+
+**A new field called "provider" is added to the TIMDEX data model** (i.e., `TimdexRecord.institution`). This field will denote--as its name implies--the institution or organization that provides _access_ to the resource described in the TIMDEX record. This field exists at the top level of the TIMDEX data model, making it easily accessible for referencing in the UI or querying. This solution also avoids using existing fields (discussed below) in ways that obscure the TIMDEX data model.
```

Comment:
I think `TimdexRecord.institution` should be updated to `TimdexRecord.provider` here.